### ISSUE TITLE
SWS/S&M: Trigger next/previous preset for ... actions do not work for some plugins

### ISSUE THREAD

**Issue by dydyczp** [reaper-oss/sws#885]
For some plugins these actions seem to not work very well.  

For these problematic plugins when some preset is loaded and the action is fired for the first time plugin state is changed according to next/previous preset but the preset combobox is not updated to the next/previous preset name (it shows "No preset") and therefore when such an action is fired second time nothing happens. 

It does not work for example (tested on some few more):
* ReaEQ
* ReaComp
* ReaDelay
* NI FM8

It does work for:
* Reasynth <- This was the only one I found

Tested on:
* SWS 2.6.9
* Reaper 5.40

**Comment by dydyczp** [comment]
Probably Reaper API bug. 

Following lua script:

```
track = reaper.GetSelectedTrack(0,0)
reaper.TrackFX_NavigatePresets(track, 0, 1)
```

reveals exactly same issue.

**Comment by swstim** [comment]
I forwarded this on to Cockos and they've fixed for v5.50rc22.
